docs/commands.md

```diff
@@ -11,26 +11,28 @@ For the most part, both axes on the ODrive can be controlled independently.
- 1. `AXIS_STATE_STARTUP_SEQUENCE` Run the [startup procedure](#startup-procedure).
- 1. `AXIS_STATE_FULL_CALIBRATION_SEQUENCE` Run motor calibration and then encoder offset calibration (or encoder index search if `<axis>.encoder.use_index` is `True`).
- 1. `AXIS_STATE_MOTOR_CALIBRATION` Measure phase resistance and phase inductance.
+ 2. `AXIS_STATE_STARTUP_SEQUENCE` Run the [startup procedure](#startup-procedure).
+ 3. `AXIS_STATE_FULL_CALIBRATION_SEQUENCE` Run motor calibration and then encoder offset calibration (or encoder index search if `<axis>.encoder.use_index` is `True`).
+ 4. `AXIS_STATE_MOTOR_CALIBRATION` Measure phase resistance and phase inductance of the motor.
     * To store the results set `<axis>.motor.config.pre_calibrated` to `True` and [save the configuration](#saving-the-configuration). After that you don't have to run the motor calibration on the next start up.
     * This modifies the variables `<axis>.motor.config.phase_resistance` and `<axis>.motor.config.phase_inductance`.
- 1. `AXIS_STATE_SENSORLESS_CONTROL` Run sensorless control.
+ 5. `AXIS_STATE_SENSORLESS_CONTROL` Run sensorless control.
     * The motor must be calibrated (`<axis>.motor.is_calibrated`)
     * [`<axis>.controller.control_mode`](#control-mode) must be `True`.
- 1. `AXIS_STATE_ENCODER_INDEX_SEARCH` Turn the motor in one direction until the encoder index is traversed. This state can only be entered if `<axis>.encoder.config.use_index` is `True`.
- 1. `AXIS_STATE_ENCODER_OFFSET_CALIBRATION` Turn the motor in one direction for a few seconds and then back to measure the offset between the encoder position and the electrical phase.
+ 6. `AXIS_STATE_ENCODER_INDEX_SEARCH` Turn the motor in one direction until the encoder index is traversed. This state can only be entered if `<axis>.encoder.config.use_index` is `True`.
+ 7. `AXIS_STATE_ENCODER_OFFSET_CALIBRATION` Turn the motor in one direction for a few seconds and then back to measure the offset between the encoder position and the electrical phase.
     * Can only be entered if the motor is calibrated (`<axis>.motor.is_calibrated`).
     * A successful encoder calibration will make the `<axis>.encoder.is_ready` go to true.
- 1. `AXIS_STATE_CLOSED_LOOP_CONTROL` Run closed loop control.
+ 8. `AXIS_STATE_CLOSED_LOOP_CONTROL` Run closed loop control.
     * The action depends on the [control mode](#control-mode).
     * Can only be entered if the motor is calibrated (`<axis>.motor.is_calibrated`) and the encoder is ready (`<axis>.encoder.is_ready`).
 
 ### Startup Procedure
 
 By default the ODrive takes no action at startup and goes to idle immediately.
-This behavior can be changed by setting any of the following to `True`. The ODrive will sequence all enabled startup actions in the order shown below.
+In order to change what startup procedures are used, set the startup procedures
+you want to `True`. The ODrive will sequence all enabled startup actions selected
+in the order shown below.
```

Comment:
I think there is something that got cut here?

Reply:
Ah nevermind the diff made it look weird. It's just a linebreak thing. I'll tidy that up no worries.